src/containers/Login/Login.js

```diff
@@ -56,6 +55,7 @@ class Login extends Component {
         if (result.status === "200") {
           auth.setItem("uuid", result.data.user.uuid);
           auth.setItem("token", result.data.token);
+          auth.setItem("about", result.data.user.about);
```

Comment:
can't we use just one object to set the whole user instead of right this one by one?

Reply:
Saved data in local Storage in one line instead of several lines.